modules/op-entitlements-shared-csi-driver.adoc

```diff
@@ -0,0 +1,123 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/configuring-security-context-for-pods.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="op-entitlements-shared-csi-driver_{context}"]
+= Using Red Hat entitlements by sharing the secret using the Shared Resources CSI driver operator
+
+You can set up sharing of the `etc-pki-entitlement` secret from the `openshift-config-managed` namespace to other namespaces using the Shared Resources Container Storage Interface (CSI) Driver operator. You can then configure your pipeline to use this secret for the Buildah task.
+
+.Prerequisites
+
+* You are logged on to your {OCP} cluster using the `oc` command line utility as a user with cluster administrator permissions.
+* You enabled the Shared Resources CSI Driver operator on your {OCP} cluster.
```

Comment:
Can we add a link to shared CSI driver 
I mean this one https://docs.openshift.com/container-platform/4.10/storage/container_storage_interface/ephemeral-storage-shared-resource-csi-driver-operator.html 
So that it will be easy to someone to understand about Shared CSI thingy

Reply:
Added to Additional Resources